torchtitan/models/llama/model.py

```diff
@@ -13,11 +13,12 @@
 import torch
 import torch.nn.functional as F
 from torch import nn
+from torchtitan.model_spec import BaseModelArgs, ModelProtocol
 from torchtitan.models.norms import build_norm
 
 
 @dataclass
-class ModelArgs:
+class ModelArgs(BaseModelArgs):
```

Comment:
Down the road we will have many models, like MM model. Do we want all model args to inherit this? Currently we use different model args for different model arch.

Reply:
This mainly for typing for now but also preserve the ability to introduce common model args.